packages/fxa-content-server/app/scripts/views/mixins/password-mixin.js

```diff
@@ -4,20 +4,22 @@
 
 // helper functions for views with passwords. Meant to be mixed into views.
 
-import KeyCodes from '../../lib/key-codes';
-import AuthErrors from '../../lib/auth-errors';
 import showPasswordTemplate from 'templates/partial/show-password.mustache';
+import AuthErrors from '../../lib/auth-errors';
+import KeyCodes from '../../lib/key-codes';
 const t = (msg) => msg;
 
+export const PASSWORD_TOGGLE_LABEL = 'label[for^="show-"]';
```

Comment:
I'm not particularly excited about a couple of the selectors that I had to use but it felt like a step in the right direction for a React refactor to not base these on classnames; we're using `#password` and `#vpassword` for the input IDs, then `#show-password`, and `#show-vpassword` for the checkboxes that show/hide the password.